misaligned.js

```diff
@@ -1,16 +1,37 @@
 const {expect} = require('chai')
+const colorPairMap = new Map();
+const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
+const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
 
-function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+const constructColorMapping = () => {
+    for (let i = 0; i <= majorColors.length; i++) {
+        for (let j = 0; j <= minorColors.length; j++) {
+            if(isIndexInRange(i, j)) {
+               colorPairMap.set((i * 5 + j)+1, {majorColor: majorColors[i], minorColor: minorColors[j]});

```

Comment:
5 is a magic number. Since the number of minor colors is 5, this is not a problem. But once the number of minor colors change from 5 to a different number, the code will have error output. There should be a test case that check the boundary values for example pair number 1, 5, 6 11 etc...